### PR TITLE
initial draft of Dopplizing-a-library document

### PR DESCRIPTION
**Pull request by commonsguy**
Note that two images are shared with the adding-Doppl-to-a-project document, so those will not show up in this topic branch.

### CODE REVIEW

File: docs/DopplLibrary.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+To do this:
+
+- Start Xcode.
+
+- If you come up to the welcome dialog, choose "Create a new Xcode project"
+(see below screenshot). If you were already in one Xcode project, choose
+File > New > Project... from the main menu.
+
+![Xcode Welcome Dialog](./AddingDoppl-4.png)
```
Comment (piannaf) [inline docs/DopplLibrary.md:289]: this is showing up as a link instead of an image when I view the readme preview

File: docs/DopplLibrary.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+- In the template selection pane of the new-project wizard, choose
+"Single View App". Then click Next.
+
+![Xcode New-Project Dialog](./AddingDoppl-5.png)
```
Comment (piannaf) [inline docs/DopplLibrary.md:294]: this is showing up as a link instead of an image when I view the readme preview

File: docs/DopplLibrary.md [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+
+Double-click on the `iosServiceApiTest.xcworkspace` file in your
+`iosServiceApiTest/` project or otherwise arrange to open it in Xcode. Once
+there, run the project, to confirm that we have no build errors and to set up
+code-completion for the rest of this tutorial.
+
+## Step #10: Testing the Library
+
+In `serviceApi/build/j2objcSrcGenTest/`, you should find a `dopplTests.txt`
+file, as before (`here is no `prefix.properties` file, because we are not
```
Comment (piannaf) [inline docs/DopplLibrary.md]: backticks got messed up here

**Review by piannaf — CHANGES_REQUESTED** [review]

File: docs/DopplLibrary.md (hunk 1 above)
Comment (commonsguy) [reply docs/DopplLibrary.md:289]: That image is shared with the adding-Doppl-to-a-project document, so it is not showing up in this topic branch, as that's in a different topic branch.

File: docs/DopplLibrary.md (hunk 2 above)
Comment (commonsguy) [reply docs/DopplLibrary.md:294]: Same thing: that image is shared with the adding-Doppl-to-a-project document, so it is not showing up in this topic branch, as that's in a different topic branch.

**Review by piannaf — APPROVED** [review]

**Pull request merged**